content/learn/book/getting-started/ecs/_index.md

```diff
@@ -144,3 +144,34 @@ hello Zayna Nieves!
 Marvelous!
 
 **Quick Note**: "hello world!" might show up in a different order than it does above. This is because systems run in parallel by default whenever possible.
+
+If we want to change the names of some people (perhaps they got married!), for example, we can do this using a mutable query:
+
+```rs
+fn update_people(mut query: Query<&mut Name, With<Person>> {
+    for mut name in &mut query {
+        if name.0 == "Elaine Proctor" {
+            name.0 = "Elaine Hume".to_string();
+            break; // We break out of the loop here, because we don’t need to change any other names
+        }
+    }
+}
+```
+
+The `query` argument must be mutable and the components we want to change have to be a mutable reference (`&mut`) for this to work.
+
+Don’t forget to add the system to the {{rust_type(type="struct", crate="bevy_app", name="Update")}} schedule:
+
+```rs
+fn main() {
+    App::new()
+        .add_systems(Startup, add_people)
+        .add_systems(Update, hello_world)
+        .add_systems(Update, (update_people, greet_people).chain())
+        .run();
+}
+```
+
+Note that we have used `.chain()` on the two systems. This is because we want them two to run in exactly the order they're listed in the code: with `update_people` occurring before `greet_people`.
+If they weren’t, the name might change after we greet the people.
+But we don’t add the `hello_world` system to this chain, because it doesn’t matter when it runs. That way Bebmvy can run `hello_world` while the other systems are running.
```

Comment:
```suggestion
But we don’t add the `hello_world` system to this chain, because it doesn’t matter when it runs. This way, bevy can run `hello_world` while the other systems are running.
```